src/common/UnkAreaFunc.inc.c

```diff
@@ -0,0 +1,11 @@
+#include "common.h"
+
+ApiStatus N(UnkAreaFunc)(ScriptInstance* script, s32 isInitialCall) {
```

Comment:
You can name this and the file `GetModelPos` or something, since that's what it obviously does